policykit/integrations/slack/models.py

```diff
@@ -423,37 +422,21 @@ class Meta:
 class SlackStarterKit(StarterKit):
     def init_kit(self, community, creator_token=None):
         for policy in self.genericpolicy_set.all():
-            if policy.is_constitution:
-                p = ConstitutionPolicy()
-                p.community = community
-                p.filter = policy.filter
-                p.initialize = policy.initialize
-                p.check = policy.check
-                p.notify = policy.notify
-                p.success = policy.success
-                p.fail = policy.fail
-                p.description = policy.description
-                p.name = policy.name
-
-                proposal = Proposal.objects.create(author=None, status=Proposal.PASSED)
-                p.proposal = proposal
-                p.save()
-
-            else:
-                p = PlatformPolicy()
-                p.community = community
-                p.filter = policy.filter
-                p.initialize = policy.initialize
-                p.check = policy.check
-                p.notify = policy.notify
-                p.success = policy.success
-                p.fail = policy.fail
-                p.description = policy.description
-                p.name = policy.name
-
-                proposal = Proposal.objects.create(author=None, status=Proposal.PASSED)
-                p.proposal = proposal
-                p.save()
+            p = Policy()
+            p.kind = Policy.CONSTITUTION if policy.is_constitution else Policy.PLATFORM
```

Comment:
@rickygv99 heads up this is likely going to conflict with your starterkit work. I think you were planning to remove the `GenericPolicy` construct, and just have start kits create new `ConstitutionPolicy` and `PlatformPolicy` records. If _this_ PR goes in, that would change yet again so that starterkits simply create `Policy` records. The field `kind` designates whether the policy is a constitution or platform policy.